docs/install.rst

```diff
@@ -27,4 +27,27 @@ Installing from GitHub
 
 .. include:: ../README.rst
    :start-after: quick-install-start-marker-do-not-remove
-   :end-before: quick-install-end-marker-do-not-remove
\ No newline at end of file
+   :end-before: quick-install-end-marker-do-not-remove
+
+.. _section-installing-from-ngc:
+
+Install from NGC Container
+--------------------------
+
+PyProf is available in the PyTorch container on the `NVIDIA GPU Cloud (NGC) 
+<https://ngc.nvidia.com>`_.
+
+Before you can pull a container from the NGC container registry, you
+must have Docker and nvidia-docker installed. For DGX users, this is
+explained in `Preparing to use NVIDIA Containers Getting Started Guide
+<http://docs.nvidia.com/deeplearning/dgx/preparing-containers/index.html>`_.
+For users other than DGX, follow the `nvidia-docker installation
+documentation <https://github.com/NVIDIA/nvidia-docker>`_ to install
+the most recent version of CUDA, Docker, and nvidia-docker.
+
+After performing the above setup, you can pull the Triton container
+using the following command::
+
+  docker pull nvcr.io/nvidia/pytorch:20.07-py3
```

Comment:
Is 20.09 the first release where this info is included? If so, should we use 20.09 here?

Reply:
We cannot use a container in master with instructions to a NGC container that not available. Someone following these instructions should get a valid container. As such, master will always point to the last NGC release container. 

This will be updated to the appropriate container for the release branch.